th/Economics and Ethics of Private Property Studies in Political Economy and Philosophy/03_p01_ch01.md

```diff
@@ -1,19 +1,26 @@
 ## Fallacies of the Public Goods Theory and the Production of Security
+## ทฤษฎีเกี่ยวกับสินค้าสาธารณะและการผลิตความมั่นคง
```

Comment:
Hi! I will try to find someone to review - but you should remove the original english that you have translated :D thank you!